libs/ledger-live-common/src/exchange/swap/updateAccountSwapStatus.ts

```diff
@@ -21,7 +21,8 @@ const maybeGetUpdatedSwapHistory = async (
     }
 
     if (pendingSwapIds.length) {
-      const newStatusList = await getMultipleStatus(pendingSwapIds);
+      const uniquePendingSwapIds = new Map(pendingSwapIds.map(item => [item.swapId, item]));
+      const newStatusList = await getMultipleStatus(Array.from(uniquePendingSwapIds.values()));
```

Comment:
we should log an error & clean duplications from the user's app.json.
Otherwise we'll be blind to the root issue being already fixed, or still there, or the next one.